audience_manager/api.rst

```diff
@@ -0,0 +1,213 @@
+.. highlight:: js
+.. default-domain:: js
+
+Audience Manager JavaScript API
+===============================
+This API provides JavaScript access to multiple resources.
```

Comment:
What actually means "access to multiple resources"? It's not clear to me. The API, in general, allows to push and read profile related data and we should communicate that.